### PR TITLE
twtxt: reinit at 1.2.3

### PR DESCRIPTION
**Pull request by siraben**
This should bring it in line with other package managers:

https://repology.org/project/twtxt/versions

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](./CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 125764` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>twtxt</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125764` at 3975b4aa run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>twtxt</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/applications/networking/twtxt/default.nix:27:0:
```
   |
27 |     description = "Decentralised, minimalist microblogging service for hackers";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/applications/networking/twtxt/default.nix [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+    substituteInPlace setup.py \
+      --replace 'aiohttp>=2.2.5,<3' 'aiohttp' \
+      --replace 'click>=6.7,<7' 'click' \
+      --replace 'humanize>=0.5.1,<1' 'humanize'
+  '';
 
-  subPackages = [ "cmd/twt" "cmd/twtd" ];
+  propagatedBuildInputs = [ aiohttp python-dateutil humanize click ];
+
+  checkInputs = [ pytest tox ];
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/twtxt/default.nix:24]:
```suggestion
  checkInputs = [ pytestCheckHook tox ];
```

File: pkgs/applications/networking/twtxt/default.nix [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
 
   meta = with lib; {
-    description = "Self-hosted, Twitter-like decentralised microblogging platform";
-    homepage = "https://github.com/jointwt/twtxt";
+    description = "Decentralised, minimalist microblogging service for hackers";
+    homepage = "https://github.com/buckket/twtxt";
     license = licenses.mit;
     maintainers = with maintainers; [ siraben ];
+    platforms = platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/twtxt/default.nix:31]:
```suggestion
```

Python sets platforms to a subset of unix.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 125764` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>twtxt</li>
  </ul>
</details>

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 125764` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>twtxt</li>
  </ul>
</details>

**Pull request merged**